scripts/webpack/worker.js

```diff
@@ -113,6 +113,7 @@ function addESNextSettings(config) {
 
 module.exports = (env = {}) => {
   let config = CONFIG;
+  env.dev = true;
```

Comment:
@ibgreen How does one produce production workers with this line? I removed it because the draco workers were always being built in dev mode.

Reply:
Yes, looks like there is a build issue in prod mode. the babel runtimes are not included. We'd need to fix that to enable prod workers.

Reply:
OK, is this tracked in a separate issue? Need to remember to remove this line...